cloudslang-compiler/src/test/resources/task-args-one-liner/flow_expression_contains_comma.yaml

```diff
@@ -18,7 +18,7 @@ flow:
   workflow:
     - CheckWeather:
         do:
-          ops.check_Weather: city = city_name, country = str('Wonderland\,Land')
+          ops.check_Weather: city = ${ city_name }, country = ${ 'Wonderland\,Land' }
```

Comment:
I think that `'Wonderland\,Land'`should remain a string to keep the original functionality